Cargo.lock

```diff
@@ -2455,14 +2481,26 @@ dependencies = [
  "syn 1.0.33",
 ]
 
+[[package]]
+name = "tokio-rustls"
+version = "0.12.3"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "3068d891551949b37681724d6b73666787cc63fa8e255c812a41d2513aff9775"
+dependencies = [
+ "futures-core",
+ "rustls 0.16.0",
+ "tokio",
+ "webpki",
+]
+
 [[package]]
 name = "tokio-rustls"
 version = "0.13.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "15cb62a0d2770787abc96e99c1cd98fcf17f94959f3af63ca85bdfb203f051b4"
 dependencies = [
  "futures-core",
- "rustls",
+ "rustls 0.17.0",
```

Comment:
I think it means we got two different versions of `tokio-rustls` and `rustls`

Reply:
Yeah I found that weird too but there does not seem to be an easy workaround.
IIRC (from last night) reqwest requires the latest version of tokio-rustls, and warp is a little behind in updating it's dependencies...